src/bundle/views/BundleDetail.tsx

```diff
@@ -368,6 +383,127 @@ const BundleDetail: FunctionComponent<BundleDetailProps> = ({ history, location,
 		});
 	}
 
+	const renderActions = () => {
+		if (isMobileWidth()) {
```

Comment:
Als de screen size verandert (bv. portrait naar landscape) dan gaat deze niet updaten.
Weet nu niet of dat echt een specifieke use case is?

Reply:
klopt, mogelijks kunnen we dit oplossen door een onDeviceRotationHook te voorzien ofzo, maar voorlopig denk ik niet dat het erg belangrijk is.